### ISSUE TITLE
UMLS reulsts

### ISSUE THREAD

**Issue by NoviceCrom** [Neph0s/LMKE#10]

**Comment by NoviceCrom** [comment]
Could you share your best hyper-parameters on UMLS like learning_rate etc?

**Comment by Neph0s** [comment]
> With
BERT-base, we set the learning rate of PLMs and other components as 10−5
and 5×10−4
respectively. For triple classification, we set the batch size as 16 and sample 1 negative triple
for each positive triple.
